lib/srv/desktop/tdp/proto.go

```diff
@@ -95,22 +95,11 @@ type byteReader interface {
 }
 
 func readRaw(in byteReader) ([]byte, error) {
```

Comment:
I think we can get rid of readRaw completely now. 

It only existed to prevent a PNG decode-encode in the proxy. Which no longer happens since the Go code never has decided data anymore.